core/includes/ajax.inc

```diff
@@ -318,6 +318,10 @@ function ajax_render($commands = array()) {
  *   @endcode
  */
 function ajax_get_form() {
+  // Exit early if $_POST is empty.
+  if (empty($_POST)) {
```

Comment:
One thought: form_build_id is essential in this function.

What if something gets posted (garbage stuff), but the form_build_id is still missing. Shouldn't we also check for that?

```
  if (empty($_POST) || !isset($_POST['form_build_id'])) {
    backdrop_exit();
  }
```
@alanmels what do you think?

Reply:
I've run checks many times and it's either `$_POST` is empty or it always contains `form_build_id`, so I thought checking the `$_POST` was sufficient. Can you confirm my finding?

Reply:
I'm not sure, it was merely an idea based on the fact that nothing can be done in that function without the "form_build_id".

I'll leave it to you, to check for it or not. Checking for an empty $_POST will for sure fix most problems anyway.

Reply:
Consider this discussion as resolved. :smile:

Reply:
Made the change, thanks for reviewing!